meshtal_analysis.py

```diff
@@ -209,11 +199,27 @@ def pick_point(x, y, z, mesh, erg):
     return v
      
 
-
-# TODO:
-def add_mesh(mesh1, mesh2):
-    """ """
-    print("not ready yet")
+def add_mesh(mesh1, mesh2): 
+    """ checks if boundaries of two meshes are equal and adds their values and errors  """
+    if mesh1.x_bounds != mesh2.x_bounds and mesh1.y_bounds != mesh2.y_bounds and mesh1.z_bounds != mesh2.z_bounds:

```

Comment:
i think this should be or rather than and.  
if any dimension bounds are not a match then it is a different spatial mesh and it is not sensible to just add it together